packages/inspector-extension/src/index.ts

```diff
@@ -87,17 +89,15 @@ const inspector: JupyterFrontEndPlugin<IInspector> = {
       return inspector;
     }
 
-    // Add command to registry.
-    commands.addCommand(command, {
-      caption: trans.__(
-        'Live updating code documentation from the active kernel'
-      ),
+    // Add inspector:open command to registry.
+    commands.addCommand(CommandIDs.open, {
+      caption,
       isEnabled: () =>
         !inspector ||
         inspector.isDisposed ||
         !inspector.isAttached ||
         !inspector.isVisible,
-      label,
+      label: () => trans.__('Show Contextual Help'),
```

Comment:
We can simplify it by removing the anonymous function:

```suggestion
      label: trans.__('Show Contextual Help'),
```